src/include/wt_internal.h

```diff
@@ -516,6 +518,7 @@ typedef uint64_t wt_timestamp_t;
 #include "btmem.h"
 #include "btree.h"
 #include "cache.h"
+#include "../evict/evict.h"
```

Comment:
Can we add the `src` directory in the standard include search path and use `#include <evict/evict.h>`?

Reply:
Yes, that's a good idea. Given that we are moving away for `include/` it makes sense. I will leave it up to @ajmorton as he is planning on moving the header file from include/ to evict/ and I will be moving it back soon.

Reply:
SGTM 👍 I'll take this up in the prototypes.py PR